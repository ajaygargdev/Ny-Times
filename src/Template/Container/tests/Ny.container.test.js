/**
 * @jest-environment jsdom
 */

import React from "react";
import { render } from "@testing-library/react";
import NyContainer from "../Ny.container";

const storeWrapper = (Comp) => () => {
  return (
    <Provider store={{}}>
      <Comp />
    </Provider>
  );
};

describe("Container Component", () => {
  test("renders without crashing", () => {
    const { container } = render(storeWrapper(NyContainer));
    expect(container).toMatchSnapshot();
  });
});
