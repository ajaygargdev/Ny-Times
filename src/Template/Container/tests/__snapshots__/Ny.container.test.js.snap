// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Container Component renders without crashing 1`] = `<div />`;
